xarray/testing/testing.py

```diff
@@ -14,15 +14,6 @@
 from xarray.core.indexes import Index, PandasIndex, PandasMultiIndex, default_indexes
```

Comment:
having a module named `testing.testing` feels a bit weird to me. Do you think `testing.assertions` would be better? The imports obviously wouldn't change since this is imported into the `testing` namespace.